x-pack/plugins/security_solution/public/detections/components/rules/step_rule_actions/rule_snooze_section.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { css } from '@emotion/react';
+import { EuiFlexGroup, EuiFlexItem, EuiText, useEuiTheme } from '@elastic/eui';
+import { RuleSnoozeBadge } from '../../../../detection_engine/components/rule_snooze_badge';
+import * as i18n from './translations';
+
+interface RuleSnoozeSectionProps {
+  id: string; // Rule SO's id (not ruleId)
+}
+
+export function RuleSnoozeSection({ id }: RuleSnoozeSectionProps): JSX.Element {
+  const { euiTheme } = useEuiTheme();
+
+  return (
+    <section>
+      <EuiText size="s">{i18n.RULE_SNOOZE_DESCRIPTION}</EuiText>
+      <EuiFlexGroup
+        alignItems="center"
+        css={css`
+          margin-top: ${euiTheme.size.s};
+        `}
+      >
+        <EuiFlexItem grow={false}>
+          <RuleSnoozeBadge id={id} showTooltipInline />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <EuiText size="s">
+            <strong>{i18n.SNOOZED_ACTIONS_WARNING}</strong>
+          </EuiText>
+        </EuiFlexItem>
```

Comment:
Looks like we show the `Actions will not be preformed until it is unsnoozed` message unconditionally, i.e. regardless of whether the rule is snoozed or not. Since it's in bold it feels like a warning in the case where it doesn't really matter:

<img width="703" alt="Screenshot 2023-04-24 at 18 36 31" src="https://user-images.githubusercontent.com/7359339/234060523-fe9161a1-0e83-4d39-a193-81c946d95106.png">

Can we hide it when the rule is not snoozed? If it's not trivial, can we make it look less dangerous by making the font regular and playing with the copy a little bit? E.g. `If snoozed actions will not be triggered`.

Reply:
@banderror your comment is addressed by this [PR](https://github.com/elastic/kibana/pull/155850).